### DISCUSSION TITLE
Sentry not capturing the performance of the queue jobs and console commands

### DISCUSSION THREAD

**Discussion by agilan-selvam-zoomrx** [getsentry/sentry-laravel#856] [Q&A] [answered]
Hello Everyone,

I have started exploring Sentry for tracking the performance and profiling of my Laravel application. While checking it out, I noticed that the performance of the queries or methods that are getting executed as part of queue jobs or console commands is getting tracked by sentry. Is it the limitation or am I missing anything with respect to configurations to track the same?

Thanks in advance.

**Comment by stayallive** [comment, marked as answer]
For queue jobs you can set [`SENTRY_TRACE_QUEUE_ENABLED=true`](https://github.com/getsentry/sentry-laravel/blob/2c70e34ee8227559b4710602409f94658f0d989a/config/sentry.php#L67-L68) in your `.env` to enable that.

We don't have it implemented for console commands since there are a lot of console commands that are not needed to be traced or are very long running (`php artisan queue:work` for example) and we haven't figured out a good way to select which should and should not be traced.

**Reply by agilan-selvam-zoomrx** [reply]
Thank for the response @stayallive . This helps.

**Reply by erik-braaksma-digipharma** [reply]
Just to add to this. I don't know the reason but we had this variable always on false and we still had job transactions.
After updating our project to Laravel 11 we started to notice that our job transactions weren't send to Sentry and that is when I found this post. Setting this to true fixed our problem.

**Reply by arasosman** [reply]
I have same problem like erik mentioned. all config true but nothing in sentry. I've tried so many things. There's also this: I see data occasionally, but only for an hour a day. I know it's ridiculous, but I couldn't find anything.

**Reply by stayallive** [reply]
> There's also this: I see data occasionally, but only for an hour a day.

I suspect this is a issue with quota's setup in your organization. I can't think of another way why it would only work for an hour a day. Please open up a new issue (or support ticket) to get to the bottom of this though.
